### DISCUSSION TITLE
Welcome to yt_videos_list Discussions!

### DISCUSSION THREAD

**Discussion by shailshouryya** [shailshouryya/yt-videos-list#9] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hapesurya** [comment]
Thanks for your great work...

Hopefully you also can add the video tutorial how to use this script. Thanks

**Reply by shailshouryya** [reply]
Hi hapesurya,

That's a good suggestion, I didn't really consider that! I'll try to add a video when I get the chance, but I haven't made videos before and I usually only work on this in my free time, so it might be a while until I get around to doing that.

In the meantime, the [python README](https://github.com/Shail-Shouryya/yt-videos-list/blob/main/python/README.md) should be enough to get you started with the basics. I know it's pretty long, but I wanted to include step by step instructions on how to get started along with working examples, so it ended up becoming way longer than I intended.

I also usually learn much better from videos than other things (like books and blogs), so I definitely can see the benefit of having a complementary video to allow users to choose between the video or the README based on their preferences when getting started. As I mentioned earlier though, it will probably be a while until I can realistically make a video for this - if you have the time and want to work on a video (no need to do this if you don't have time or get stuck with something), feel free to upload it somewhere like YouTube and reply to this comment here, and I'll link your video in the documentation! :)
